messaging/mqtt/pubsub.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mqtt
+
+import (
+	"errors"
+	"time"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+)
+
+const (
+	protocol = "mqtt"
+	id       = "mqtt-publisher"
+	qos      = 1
+)
+
+var errConnect = errors.New("failed to connect to MQTT broker")
+
+func newClient(address string, timeout time.Duration) (mqtt.Client, error) {
+	opts := mqtt.NewClientOptions().
+		AddBroker(address).
+		SetUsername(id).
+		SetPassword(id).
+		SetClientID(id).
+		SetCleanSession(false)
+	client := mqtt.NewClient(opts)
+	tkn := client.Connect()
```

Comment:
What does `tkn` stand for?

Reply:
It stands for `token`. That's the construct Paho internally returns on the async operations.

Reply:
Why not `token`?

Reply:
It is bit confusing IMHO, maybe comment could help?

Reply:
@drasko why a comment?

Reply:
why not `token` than?